gui.ml

```diff
@@ -1369,7 +1369,7 @@ are immediately sent to jail.\n" in
     let help_str_lst = String.split_on_char '\n' help_message in
     (* this state is a dummy state *)
     let test_state = State.(init_state
-      (Yojson.Basic.from_file "test_board.json") ["a", GUI] ~random:true) in
+      (Yojson.Basic.from_file "boards/test_board.json") ["a", GUI] ~random:true) in
```

Comment:
In 3110 A3, we used Filename.dir_sep to avoid hardcoding a backslash if you remember. From the A3 writeup:
> Anywhere you need to use a path separator in your code, use Filename.dir_sep rather than hard-coding a forward or backward slash; this will ensure that your code works on Windows and Unix platforms."

Could you make this change and not hard code the backslash? Should be something like `"boards"^Filename.dir_sep^"test_board.json"`.